openssh/README.md

```diff
@@ -0,0 +1,132 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+set -x
```

Comment:
I suggest keeping the instructions to an absolute minimum to help the reader digest them - maybe this line can be removed.